src/lib/web-component/filter-option/filter-option-element.js

```diff
@@ -49,6 +51,8 @@ export class FilterOption extends HTMLElement {
   }
 
   addNewCheckbox(checkboxLabel) {
+    this.emptyFilterLabel.hidden = true;
```

Comment:
This can be done with pure CSS:

- Either with the element, by letting the element be the last child and using the `+` selector to target the last label if it's hidden.
- Or with the `:empty` selector and pseudo-elements.

... or by just removing the filter content at the first run.

Currently, if the filter doesn't match anything, the table is still empty. Is that intentional?

Reply:
I used the `:empty` selector and pseudo-elements to fix this.